ledger-core/v2/application/testwalletapi/handlers.go

```diff
@@ -140,6 +140,53 @@ func (s *TestWalletServer) Transfer(w http.ResponseWriter, req *http.Request) {
 		Error:   "",
 	}
 	defer func() { s.mustWriteResult(w, result) }()
+
+	fromRef, err := insolar.NewReferenceFromString(params.From)
+	if err != nil {
+		result.Error = fmt.Sprintf("Failed to create reference from string (%s), sender reference", params.From)
```

Comment:
probably we should log original error

Reply:
I think it would be useful in error cases

Reply:
Fixed